httprange.go

```diff
@@ -36,93 +39,77 @@ func (e InvalidRange) Error() string {
 
 // HttpRange specifies the byte range to be sent to the client.
 type httpRange struct {
```

Comment:
for HTTP range, as per [rfc7233](https://tools.ietf.org/html/rfc7233) it calls first/last BytePos which is more appropriate I think.